make_mesh.py

```diff
@@ -0,0 +1,199 @@
+import sys
+from glob import glob
+import skimage
+from skimage import measure
+import nibabel
+import numpy
+import os
+from math import floor
+import scipy
+import argparse
+#
+def remove_inner_surface(img_data,mask,treshhold=0):
+	"""
+	Function to replace inner data of the given volume with a smoothed, uniform masking to avoid generation of inner surface structures and staircase artifacts when using marching cube algorithm
+	
+	Input:
+	img_data :input volume data to extract mesh from, numpy-array
```

Comment:
please follow the numpy docstring guidelines: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html